tiledb/sm/cpp_api/array_schema.h

```diff
@@ -645,6 +646,34 @@ class ArraySchema : public Schema {
     return has_attr == 1;
   }
 
+  /**
+   * Checks if the schema has a dimension label of the given name.
+   *
+   * @param name Name of dimension label to check for.
+   * @return True if the schema has a dimension label of the given name.
+   */
+  bool has_label(const std::string& name) const {
+    auto& ctx = ctx_.get();
+    int32_t has_label;
+    ctx.handle_error(tiledb_array_schema_has_dimension_label(
+        ctx.ptr().get(), schema_.get(), name.c_str(), &has_label));
+    return has_label == 1;
+  }
```

Comment:
`array_schema_experimental.h` already has a `has_dimension_label` method. It is intentionally not in this function to avoid mixing the experimental and non-experimental API.